controllers/imagerepository_controller.go

```diff
@@ -231,6 +231,51 @@ func (r *ImageRepositoryReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	return ctrl.Result{}, nil
 }
 
+func (r *ImageRepositoryReconciler) AddNotifications(ctx context.Context, imageRepository *imagerepositoryv1alpha1.ImageRepository) ([]imagerepositoryv1alpha1.NotificationStatus, error) {
+	log := ctrllog.FromContext(ctx).WithName("ConfigureNotifications")
+	log.Info("Configuring notifications", "Name", imageRepository.Name, "Namespace", imageRepository.Namespace)
+
+	if imageRepository.Spec.Notifications == nil {
+		// No notifications to configure
+		return nil, nil
+	}
+
+	notificationStatus := []imagerepositoryv1alpha1.NotificationStatus{}
+
+	for _, notification := range imageRepository.Spec.Notifications {
+		log.Info("Creating notification in Quay", "Title", notification.Title, "Event", notification.Event, "Method", notification.Method)
+		quayNotification, err := r.QuayClient.CreateNotification(
+			r.QuayOrganization,
+			imageRepository.Spec.Image.Name,
+			quay.Notification{
+				Title:  notification.Title,
+				Event:  notification.Event,
+				Method: notification.Method,
+				Config: quay.NotificationConfig{
+					Url: notification.Config.Url,
+				},
+				EventConfig: quay.NotificationEventConfig{},
+			})
+		if err != nil {
+			log.Error(err, "failed to send notification", "Title", notification.Title, "Event", notification.Event, "Method", notification.Method)
```

Comment:
Word `send` is confusing here, better say `create` or `setup`.

Reply:
Fixed